pkg/local_object_storage/engine/inhume.go

```diff
@@ -73,7 +80,7 @@ func (e *StorageEngine) inhume(addr *objectSDK.Address, prm *shard.InhumePrm, ch
 				zap.String("error", err.Error()),
 			)
 		} else {
-			res = new(InhumeRes)
+			ok = true
 		}
 
 		return err == nil
```

Comment:
I think you need `return` there instead of `return err == nil` right now.

Reply:
We inhume the objects in the "best" shard. so we need to stop iterator by return `err == nil`.

Reply:
Oh I missed that this return statement inside shard iterator. Got it.